nengo_spa/algebras/vtb_algebra.py

```diff
@@ -112,32 +112,34 @@ def fractional_bind(self, v, exponent):
 
         sub_d = self._get_sub_d(len(v))
         sub_m = fractional_matrix_power(
-            np.sqrt(sub_d) * v.reshape((sub_d, sub_d)), exponent
+            np.sqrt(sub_d) * v.reshape((sub_d, sub_d)).T, exponent
         )
         m = np.kron(np.eye(sub_d), sub_m)
         # more efficient than the equivalent:
         # m = fractional_matrix_power(self.get_binding_matrix(v), exponent)
         return m.dot(self.identity_element(len(v)))
 
     def make_nondegenerate(self, v):
-        # note: these "Householder vectors" are not only unitary, but symmetric
+        # nondegenerate vectors are unitary and
+        # form a matrix with a positive determinant
```

Comment:
This seems to be wrong? `np.linalg.det(-np.eye(d))` is 1, but `fractional_matrix_power(-np.eye(d), 0.5)` gives complex values.
I think the relevant property for (T)VTB is positive definiteness.